appsec/config_test.go

```diff
@@ -0,0 +1,243 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package appsec
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAPISecConfig(t *testing.T) {
+	t.Run("API Security", func(t *testing.T) {
```

Comment:
I don't think this top-level `t.Run("API Security"` is very useful... I'd remove that and un-nest, since the test function name already states the same text.

Reply:
af96bef40d5e6